packaging/helm/arlas-server/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: 5.6.0
```

Comment:
Is it possible to get this value automatically or to insert it through `release.sh` process like we have already done for API_VERSION ?
If we keep it hardcoded, we will have to update this file before each release manually... and probably forget to do it ;)

Reply:
Do you think You coudl handle this before tomorrow evening? I would like to release arlas-server on friday and it would be nice to have helm packaging included.

Reply:
should be fixed by 1af5c55bf439f7e44c5cbf095a20485c03a74bc5 (see [here](https://github.com/gisaia/ARLAS-server/blob/1af5c55bf439f7e44c5cbf095a20485c03a74bc5/release.sh#L125-L126)).

Reply:
Ok, but you have to replace FULL_API_VERSION by ARLAS_VERSION for these files (basically, next ARLAS_VERSION=5.6.1 while FULL_API_VERSION=5.0.1) and you should remove hardcoded reference to `5.6.0` in your yml files to avoid confusion.
By the way, we will release helm packaging in 5.6.2. I would be safer ;)

Reply:
Should be fixed in ec502d3797f114decd0e8d89ef1d2bb0f946e724